pkg/integration/tests/bisect/basic.go

```diff
@@ -31,28 +32,29 @@ var Basic = NewIntegrationTest(NewIntegrationTestArgs{
 
 		t.Views().Commits().
 			Focus().
-			SelectedLine(Contains("commit 10")).
-			NavigateToLine(Contains("commit 09")).
+			SelectedLine(Contains("CI commit 10")).
```

Comment:
I try to avoid a Contains call with two different columns given that we could rearrange things in future. How about the more explicit `Contains("commit 10").DoesNotContain("(*)")`?

Reply:
In this particular test a `DoesNotContain` would work, but what about 59a8ed1aa174? I wouldn't want to add `DoesNotContain` calls to every line.

How likely is it that we rearrange things in the future?

Reply:
I thought about this some more. I really like the way the tests are written here, and I think I'd just take the risk of having to adapt them when we change the layout in the future. It won't be a terrible amount of work.

Reply:
Fair enough, happy to go with your approach